syntax = "proto3"; //声明proto的版本

package biz; //作用域
option cc_generic_services = true;


//登录请求
message GetFriendListsReq
{
    uint32 uid = 1;
}
message GetFriendListsAck
{
    int32 result = 1;
    repeated bytes friends = 2;
}

service FriendServiceRpc
{
    rpc GetFriendLists(GetFriendListsReq) returns(GetFriendListsAck);
}
